docs/manual_full.tex

```diff
@@ -28,6 +31,11 @@
    margin=2.54cm
 }
 
+% pandoc packages
+\usepackage{lmodern} % Is it necessary? CM looks the same, doesn't it?
```

Comment:
`lmodern` is there to make sure it's using scalable fonts, not bitmap fonts. Depending on system setup that may already be the default, this just forces it. (One should think it's not necessary any more in 2020, but ..)